zerver/views/upload.py

```diff
@@ -22,7 +23,13 @@ def serve_local(request: HttpRequest, path_id: str) -> HttpResponse:
     local_path = get_local_file_path(path_id)
     if local_path is None:
         return HttpResponseNotFound('<p>File not found</p>')
-    return sendfile(request, local_path)
+
+    attachment = True
+    file_type = guess_type(local_path)[0]
+    if 'image' in file_type or 'pdf' in file_type:
+        attachment = False
```

Comment:
Based on https://docs.python.org/2/library/mimetypes.html, I think `guess_type()[0]` can be None, which I think will throw an exception in this block.  Can you add a test for a filename with no extension?

Reply:
sure, will do